tests/bsim/bluetooth/host/misc/sample_test/src/main.c

```diff
@@ -9,11 +9,33 @@
 #include "bs_tracing.h"
 #include "bstests.h"
 #include "babblekit/testcase.h"
+#include "testlib/log_utils.h"
+
 
 extern void entrypoint_dut(void);
 extern void entrypoint_peer(void);
 extern enum bst_result_t bst_result;
 
+static uint8_t	set_log_level_run_time = 3;
+
+static void test_args(int argc, char *argv[])
+{
+	size_t argn = 0;
+	const char *arg = argv[argn];
+
+	if(strcmp(arg, "log_level") == 0){
+		set_log_level_run_time = strtoul(argv[++argn], NULL, 10);
```

Comment:
Since you are storing the result of `strtoul` directly in `set_log_level_run_time` then the type should be `unsigned long`.

Reply:
changed it to `unsigned long`